peps/pep-9999.rst

```diff
@@ -0,0 +1,126 @@
+PEP: 9999
+Title: Static description file for Python installations
+Author: Filipe Laíns <lains@riseup.net>
+PEP-Delegate:
+Status: Draft
+Type: Informational
+Content-Type: text/x-rst
+Created: 01-Oct-2023
+Python-Version: 3.13
+Resolution:
```

Comment:
Informational PEPs don't need a Python-Version, and to make the linter happy let's add Resolution back when we have one.
```suggestion
```